docs/system-admin-guide/manage-work-packages/work-package-status/README.md

```diff
@@ -19,16 +19,17 @@ To learn about the options for configuring the transition from one status to ano
 
 To create a new work package status click the green ***+ New Status*** icon. 
```

Comment:
Incorrect button label: it's "+ Status", not "+ New Status".

Reply:
Fixed.